app/src/main/java/com/aptoide/uploader/apps/InstalledAppsManager.java

```diff
@@ -33,7 +33,8 @@ public InstalledAppsManager(InstalledPersistence installedPersistence,
   }
 
   private Observable<List<InstalledApp>> getInstalledApps() {
-    return installedPersistence.allInstalledSorted();
+    return installedPersistence.allInstalledSorted()
+        .filter(list -> !list.isEmpty());
```

Comment:
What if the user does not have any apps installed? What happens? we should have an empty view for it, if we have, this will prevent it from showing right?

Reply:
We are not filtering for our own package, which means that the user will always have at least one app, so that issue will never happen.
We don't have an empty view for it, but i think we should add it in the future (also maybe filtering some packages?)